index.html

```diff
@@ -1,58 +1,64 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.com/libraries/normalizenormalize.css"
+        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@300;600&display=swap" rel="stylesheet">
+</head>
 
-    <body class="main">
-        <div class="menu">
-            <img src="./Menu.png">
-            <span class="text-menu">Foundation <br> Startup landing template</span>
-            <nav class="one">
-                <ul class="topmenu">
-                    <li><a href="" class="active">Library<span class="fa fa-angle-down"></span></a>
-                        <ul class="submenu">
-                            <div class="menu-choose">
-
-                                <div class="acquire">
-                                    <div class="aqui">Acquire USERS
-                                        <span class="aquam">Work with the tools and services you use</span>
-                                    </div>
-                                </div>
-                                <div class="engage">Engage</div>
-                                <div class="book">Book</div>
-                                <div class="general">General</div>
-                            </div>
-
-                        </ul>
-                    </li>
-                    <li class="menu-tab">Stories</li>
-                    <li class="menu-tab">Home</li>
-                </ul>
-            </nav>
-            <button href="" class="btmen">Get Started</button>
+<body>
+    <div class="footer">
+        <div class="cub1">
+            <img class="picture" src="./picture-logo.png" width="60px" height="60px">
+            <p class="name-logo">Foundation
+                <li class="under-logo">Startup landing template</li>
+                <li class="text-under-logo">Foundation is a website template
+                    for<br>startups and small teams. It helps to<br>build a website in
+                    no time.</li>
+            <div class="buttons">
+                <a class="app-store" href="#"><img class="picture-app_store" src="./app-store.png"
+                     alt="button app-store"></a>
+                <a class="google-play" href="#"><img class="picture-google_play" src="./google-play.png"
+                        alt="button google-play"></a>
+            </div>
         </div>
-
-        <div class="picture">
-            <img src="./ouch.png">
+        <div class="cub2">
+            <span class="name-product">Product</span>
+            <ul>
+                <a class="text" href="#">Acquire Users</a>
```

Comment:
`.chapter-link` is a better class name. Here and elsewhere be more specific with class names